classes/task/merge_user_accounts.php

```diff
@@ -0,0 +1,184 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+namespace tool_mergeusers\task;
+defined('MOODLE_INTERNAL') || die();
+use MergeUserTool;
```

Comment:
I think it is better and safer to put this `use MergeUserTool;` after the `require_once(... autoload.php');`, since this `autoload.php` is  the responsible to find all plugin classes, including `MergeUserTool`.

Reply:
Mmmmm... now, there is no `require_once(....autoload.php');` This could cause that the class MergeUserTool be not found. 

So, we should have something like:

```php
namespace tool_mergeusers\task;

defined('MOODLE_INTERNAL') || die;

require_once(__DIR__ . '/../../lib/autoload.php');
￼use MergeUserTool;
```

This will ensure that Moodle will ALWAYS find the class MergeUserTool.